src/app/hiv-care-lib/case-management/case-management-filters.component.ts

```diff
@@ -200,6 +242,11 @@ export class CaseManagementFiltersComponent implements OnInit, OnChanges {
         this.hasCaseManager = $event;
         this.toggleCaseManagerControl($event);
     }
+    public onCaseManagerSelected($event) {
+        this.selectedCaseManager = $event;
+        // this.getCaseManagersIds($event);
+        console.log('case manager selected', $event);
```

Comment:
Could we get rid of the debug statements?

Reply:
Its still under test. Not ready for merging yet